modules/programs/process-compose.nix

```diff
@@ -0,0 +1,33 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib) mkIf;
+  cfg = config.programs.process-compose;
+  yamlFormat = pkgs.formats.yaml { };
+in
+{
+  meta.maintainers = [ lib.hm.maintainers.bbigras ];
+  options.programs.process-compose = {
+    enable = lib.mkEnableOption "the process-compose orchestrator";
+    package = lib.mkPackageOption pkgs "process-compose" { nullable = true; };
+    settings = lib.mkOption {
+      type = yamlFormat.type;
+      default = { };
+      defaultText = lib.literalExpression "{}";
+      description = "The process-compose configuration.";
+    };
+  };
+  config = mkIf cfg.enable {
+    home.packages = mkIf (cfg.package != null) [ cfg.package ];
+
+    xdg.configFile."process-compose/settings.yaml" =
```

Comment:
Is the path the same for darwin (with or without xdg enabled) and linux? 

I see the other PR doing this https://github.com/nix-community/home-manager/pull/5788/files#diff-0250cdf0ec8bf9d6c96a28941631125be687497d95ea33f037397f1d20abd5f0R48-R51

Reply:
Thanks for the comment. I have no idea how darwin works, so I just took the same code as the other PR.